service/history/api/get_workflow_util.go

```diff
@@ -361,5 +361,6 @@ func MutableStateToGetResponse(
 		InheritedBuildId:             mutableState.GetInheritedBuildId(),
 		MostRecentWorkerVersionStamp: mostRecentWorkerVersionStamp,
 		TransitionHistory:            mutableState.GetExecutionInfo().TransitionHistory,
+		VersioningInfo:               mutableState.GetExecutionInfo().VersioningInfo,
```

Comment:
I think we might have forgotten to add this while making the versioning-3 changes - this is required for query tasks